CONTRIBUTING.md

```diff
@@ -1,4 +1,282 @@
-Contributing code to Element
-============================
+Contributing code to Element Web
```

Comment:
Basically a copy of https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md with `s/matrix-js-sdk/Element Web` .